src/resolvers/donationResolver.test.ts

```diff
@@ -1570,3 +1572,96 @@ function donationsByDonorTestCases() {
     );
   });
 }
+
+function donationsToWalletsTestCases() {
+  const walletAddress1 = generateRandomEtheriumAddress();
+  const walletAddress2 = generateRandomEtheriumAddress();
+  const walletAddress3 = generateRandomEtheriumAddress();
+  const walletAddress4 = generateRandomEtheriumAddress();
+  it('should find donations with special destination successfully', async () => {
+    const project = await saveProjectDirectlyToDb(createProjectData());
+    const user1 = await User.create({
+      walletAddress: walletAddress1,
+      loginType: 'wallet',
+      firstName: 'bacheTest',
+    }).save();
+    const accessToken1 = await generateTestAccessToken(user1.id);
+    const donation1 = await axios.post(
+      graphqlUrl,
+      {
+        query: saveDonation,
+        variables: {
+          projectId: project.id,
+          chainId: NETWORK_IDS.XDAI,
+          transactionNetworkId: NETWORK_IDS.XDAI,
+          fromAddress: walletAddress2,
+          toAddress: walletAddress1,
+          transactionId: generateRandomTxHash(),
+          amount: 10,
+          token: 'GIV',
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken1}`,
+        },
+      },
+    );
+
+    const result = await axios.post(
+      graphqlUrl,
+      {
+        query: donationsToWallets,
+        variables: {
+          toWalletAddresses: [walletAddress1],
+        },
+      },
+      {},
+    );
+    let test = true;
+    result.data.data.donationsToWallets.forEach(item => {
+      if (item.toWalletAddress !== walletAddress1) {
+        test = false;
+      }
+    });
+    assert.equal(test, true);
+  });
+  it('should find donations with special destination in uppercase successfully', async () => {
+    const result = await axios.post(
+      graphqlUrl,
+      {
+        query: donationsToWallets,
+        variables: {
+          toWalletAddresses: [walletAddress1.toUpperCase()],
+        },
+      },
+      {},
+    );
+    let test = true;
+    result.data.data.donationsToWallets.forEach(item => {
+      if (item.toWalletAddress !== walletAddress1) {
+        test = false;
+      }
+    });
+    assert.equal(test, true);
```

Comment:
Instead of what you have written, you can use this

```
  result.data.data.donationsToWallets.forEach(item => {
      assert.equal(item.toWalletAddress, walletAddress1)
    });
```